src/redux/sagas/wallet/onboard.ts

```diff
@@ -0,0 +1,76 @@
+import Onboard, { InitOptions } from '@web3-onboard/core';
+import injectedWallets from '@web3-onboard/injected-wallets';
+
+import { private_keys as ganachePrivateKeys } from '../../../../amplify/mock-data/colonyNetworkArtifacts/ganache-accounts.json';
+
+import colonyIcon from '~images/icons/colony-logo.svg';
+import {
+  isDev,
+  TERMS_AND_CONDITIONS,
+  VERSION,
+  TOKEN_DATA,
+  GANACHE_NETWORK,
+  GANACHE_LOCAL_RPC_URL,
+} from '~constants';
+import { intl } from '~utils/intl';
+import { getChainIdAsHex } from '~utils/autoLogin';
+import { Network } from '~types';
+
+import ganacheModule from './ganacheModule';
+
+const { formatMessage } = intl({
+  'metadata.name': 'ColonyCDapp',
+  'metadata.description': `An iteration of the Colony Dapp sporting both a fully decentralized operating mode, as well as a mode enhanced by a metadata caching layer`,
+});
+
+const getDevelopmentWallets = () => {
+  if (isDev) {
+    return (
+      Object.values(ganachePrivateKeys)
+        .map((privateKey, index) => ganacheModule(privateKey, index + 1))
+        /*
+         * Remove the wallets used by the reputation miner and the block ingestor
+         * As to not cause any "unplesantness"
+         */
+        .slice(0, -2)
+    );
+  }
+  return [];
+};
+
+const wallets = [injectedWallets(), ...getDevelopmentWallets()];
+
+const onboardConfig: InitOptions = {
+  wallets,
+  // Chains array only used in `ganacheModule` for use in development.
+  chains: [
```

Comment:
The onboard config only uses the `chains` array to instantiate non-injected wallet modules (like `ganacheModule`), and for programmatic chain switching. We're not doing any programmatic chain switching, and we only have one non-injected module (`ganacheModule), so we only need to include one chain (ganache) in the config.

Reply:
Are you sure? I couldn't get QA running on Gnosis without providing a chain. Granted I didn't try to just give it the local ganache chain details.